catalog.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="utf-8" />
+  <title>HTML Academy: каталог Нёрдс</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      <a href="index.html">
+        <img src="#" alt="Логотип дизайн-студии Nerds" />
+      </a>
+      <ul>
+        <li><a href="index.html">Студия</a></li>
+        <li><a href="#">Клиенты</a></li>
+        <li><a href="#">Магазин</a></li>
+        <li><a href="#">Котакты</a></li>
+      </ul>
+      <a href="#">Корзина</a>
+    </nav>
+  </header>
+  <main>
+    <h1>Магазин готовых шаблонов</h1>
+    <section>
+      <h2>Фильтр товаров</h2>
+      <div>
+        <p>сортировать:</p>
+        <ul>
+          <li>
+            <a href="#">по цене</a>
+            <a href="#">по типу</a>
+            <a href="#">по названию</a>
+            <a href="#">по убыванию</a>
+            <a href="#">по возрастанию</a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <h2>список готовой продукции</h2>
+      <ul>
+        <li>
+          <img src="#" alt="Картинка сайта для туристов в городе Седона" />
+          <div>
+            <a href="#">SEDONA</a>
```

Comment:
можно название сделать заголовком
```suggestion
            <h3><a href="#">SEDONA</a></h3>
```